apps/remote_control/lib/lexical/remote_control.ex

```diff
@@ -90,7 +90,8 @@ defmodule Lexical.RemoteControl do
   end
 
   defp app_globs do
-    Enum.map(@allowed_apps, fn app_name -> "/**/#{app_name}*/ebin" end)
+    app_globs = Enum.map(@allowed_apps, fn app_name -> "/**/#{app_name}*/ebin" end)
+    ["/**/priv" | app_globs]
```

Comment:
This was a fun discovery. When we launch the _second_ vm , it wasn't adding the priv directories to the paths.